src/strand-item-recycler/strand-item-recycler.js

```diff
@@ -548,9 +553,11 @@ found here: https://github.com/Polymer/core-list
 		},
 
 		_needsInitialization: function (data, _templateFound) {
-			this._initializable = false;
-			this._initialized = false;
-			this.initialize();
+			if (this._isAttached) {
```

Comment:
unclear about the flow internally here, but what happens in the case where it is attached some time later?